pkg/controller/volume/persistentvolume/pv_controller.go

```diff
@@ -1434,13 +1434,20 @@ func (ctrl *PersistentVolumeController) provisionClaimOperation(claim *v1.Persis
 		return
 	}
 
+	var selectedNode *v1.Node = nil
+	if nodeName, ok := claim.Annotations[annSelectedNode]; ok {
+		selectedNode, err = ctrl.kubeClient.CoreV1().Nodes().Get(nodeName, metav1.GetOptions{})
```

Comment:
I think it's better to use a node informer cache instead of an api call every time.